platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/migration/ConfigurationMigrationManager.java

```diff
@@ -128,6 +135,11 @@ public void init() throws Exception {
         return export(Paths.get(exportDirectory));
     }
 
+    @Override
+    public Collection<Describable> getInfo() {
+        return listMigratableDescribables();
```

Comment:
I'm not saying we need to change the return type from `Collection<Describable>` to `Collection<Migratable>`. I'm just saying that the internal list will contain `DataMigratable` objects but we will return it as a `Collection<Describable>`.